wgsl/index.bs

```diff
@@ -2074,6 +2074,35 @@ See [[#arithmetic-expr]].
   </xmp>
 </div>
 
+WGSL also [=predeclared|predeclares=] the following [=type aliases=]:
+
+<table class='data'>
+  <thead>
+    <tr>
+        <th>Predeclared alias
+        <th>Original type
+        <th>Restrictions
+  </thead>
+  <tr> <td><dfn noexport>mat2x2f</dfn> <td>mat2x2&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat2x3f</dfn> <td>mat2x3&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat2x4f</dfn> <td>mat2x3&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat3x2f</dfn> <td>mat3x2&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat3x3f</dfn> <td>mat3x3&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat3x4f</dfn> <td>mat3x3&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat4x2f</dfn> <td>mat2x2&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat4x3f</dfn> <td>mat2x3&lt;f32&gt; <td>
+  <tr> <td><dfn noexport>mat4x4f</dfn> <td>mat2x3&lt;f32&gt; <td>
```

Comment:
These three are all wrong: 4x2, 4x3, 4x4

These comments apply to the `h` variants too, but I won't duplicate them :-)